src/reducers/index.js

```diff
@@ -0,0 +1,36 @@
+import {
+  createStore, combineReducers, applyMiddleware, compose,
+} from 'redux';
+import reduxThunk from 'redux-thunk';
+import SetCurrentUserTokenReducer from './Setcurrentusertoken';
+import SelectedHouseReducer from './Selecthousereducer';
+import SetErrorReducer from './Seterrorreducer';
+import SetSignErrorReducer from './Setsigninerrorreducer';
+import SetCategoryReducer from './Setcategoryreducer';
+import SetStatusReducer from './setstatusreducer';
+
+const rootReducer = combineReducers({
+  currentusertoken: SetCurrentUserTokenReducer,
+  selectedhouse: SelectedHouseReducer,
+  errormessage: SetErrorReducer,
+  errorsignin: SetSignErrorReducer,
+  filter: SetCategoryReducer,
+  status: SetStatusReducer,
+});
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+
+const initialState = {
```

Comment:
In the separate reducers we are already setting initial states. Why do we need to again set them up here?